src/main/java/io/temporal/internal/replay/WorkflowContext.java

```diff
@@ -102,25 +103,25 @@ WorkflowExecution getParentWorkflowExecution() {
 
   int getWorkflowRunTimeoutSeconds() {
     WorkflowExecutionStartedEventAttributes attributes = getWorkflowStartedEventAttributes();
-    return attributes.getWorkflowRunTimeoutSeconds();
+    return (int) Durations.toSeconds(attributes.getWorkflowRunTimeout());
```

Comment:
Should we change all the methods here to return Durations instead?

Reply:
yes, with exception of get..Timestamp ones.